docs/howtos/quick-start/cli/_cli-basic-querying.mdx

```diff
@@ -0,0 +1,100 @@
+
+- Connect to Redis using CLI or **[RedisInsight](https://redis.io/docs/stack/insight/)** (a GUI tool to visualize data & run commands)
+
+**RedisInsight**
+![RedisInsight](../images/redis-insight.svg)
+
+```shell
+# syntax 1 : connect using host & port
+$ redis-cli -h host -p port -a password
+
+# example 1
+$ redis-cli -h redis15.localnet.org -p 6390 -a myUnguessablePassword
+
+# syntax 2 : connect using uri
+$ redis-cli -u redis://user:password@host:port/dbnum
+
+# example 2
+$ redis-cli -u redis://LJenkins:p%40ssw0rd@redis-16379.hosted.com:16379/0
+
+```
+
+- Basic CLI / RedisInsight workbench commands
+
+```shell
+# syntax : Check specific keys
+> keys pattern
```

Comment:
In the other files, commands are in all caps when showing CLI examples, might want to reflect that here as well